pystiche_papers/sanakoyeu_et_al_2018/_modules.py

```diff
@@ -0,0 +1,182 @@
+from typing import List, Tuple, Union, cast
+
+import torch
+from torch import nn
+
+from ..utils import ResidualBlock, same_size_padding
+
+__all__ = [
+    "get_padding",
+    "activation",
+    "conv",
+    "ConvBlock",
+    "ConvTransponseBlock",
+    "residual_block",
+]
+
+
+def get_padding(
+    padding: str, kernel_size: Union[Tuple[int, int], int]
+) -> Union[Tuple[int, int], int]:
+    if padding == "same":
+        return cast(Tuple[int, int], same_size_padding(kernel_size))
+    elif padding == "valid":
+        return 0
+    else:
+        raise ValueError
```

Comment:
Please add a descriptive error message. You could use `pystiche.misc.verify_str_arg`.